protocols/DNS4.md

```diff
@@ -0,0 +1,21 @@
+# `dns4`
+
+`dns4` is a protocol that defines which domain name should be used, but only by resolving it to a IPv4 address.
```

Comment:
```suggestion
`dns4` is a protocol that defines which domain name should be used, but only by resolving it to IPv4 addresses.
```

`dns4` can resolve to one or more ipv4 address.

Reply:
Good point. I will change that.